static/app/views/dashboardsV2/widgetBuilder/widgetBuilder.tsx

```diff
@@ -534,6 +537,14 @@ function WidgetBuilder({
         }
       }
 
+      if (widgetBuilderNewDesign) {
+        if (queryIndex === 0) {
+          newQuery.orderby = aggregateAliasFieldStrings[0];
+        }
+
+        newQuery.fieldAliases = columnsAndAggregates?.fieldAliases ?? [];
```

Comment:
Style choice, feel free to dismiss, but I would prefer to move this under line 540. The main benefit there is that we can see the default behavior first, and then the   `if(queryIndex===0)` branch, which makes the control flow slightly easier to read.

My first thought when I read this was "oh you forgot to return on L542", but you are actually setting different properties.